src/modules/Elsa.Workflows.Core/Serialization/Converters/PolymorphicObjectConverter.cs

```diff
@@ -130,9 +131,13 @@ public override object Read(ref Utf8JsonReader reader, Type typeToConvert, JsonS
             {
                 array.SetValue(deserializedElement, index++);
             }
-            else
+            else if (collection is HashSet<object> hashSet)
```

Comment:
Can we do `ISet` in order to capture more cases?

`if (collection is ISet hashSet)`

Please try it, I'm not sure it will compile or work at runtime.

Reply:
not found ISet , found ISet<T>

Reply:
Sorry, that sentence doesn’t make sense to me. Can you rephrase please?

Reply:
I understand now, GitHub removed the greater than and less than symbols when you types `Set` and `Set<>`.